backend/models/entry.js

```diff
@@ -0,0 +1,79 @@
+import Image from './image'
+import DataTypes from 'sequelize'
+import sequelize from '../config/sequelize'
+
+const IMAGE = 3
```

Comment:
Where are the magic numbers coming from?

Reply:
They're just a mimic-enum -- i'll move them to a `constants.js` file